sources/academy/webscraping/scraping_basics_python/05_parsing_html.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Parsing HTML with Python
+sidebar_label: Parsing HTML
+description: Lesson about building a Python application for watching prices. Using the Beautiful Soup library to parse HTML code of a product listing page.
+sidebar_position: 5
+slug: /scraping-basics-python/parsing-html
+---
+
+**In this lesson we'll look for products in the downloaded HTML. We'll use BeautifulSoup to turn the HTML into objects which we can work with in our Python program.**
+
+---
+
+From lessons about browser DevTools we know that the HTML tags representing individual products have a `class` attribute which, among other values, contains `product-item`.
+
+![Products have the ‘product-item’ class](./images/product-item.png)
+
+As a first step, let's try counting how many products is on the listing page.
+
+## Treating HTML as a string
```

Comment:
This feels a bit useless since it is not much simpler than using HTML parser and everyone uses HTML parser in the end

Reply:
In 458dc28d I removed the build up and just explained why having an HTML parser is a good idea. I take this as addressed, but I'll still revisit all the lessons as whole and think about whether I can cut down on explanations and how.